### PR TITLE
31 create profile pic controller

### PR DESCRIPTION
**Pull request by MasterCoookie**

### CODE REVIEW

File: middleware/uploadMiddleware.js [hunk 1]
```diff
@@ -1,29 +1,57 @@
...
+        }
+    }
+ });
+
 
  const docsUpload = multer({
-    storage: uploadStorage,
+    storage: docsUploadStorage,
+    limits: {
+        filesize: 25000000
```
Comment (macibol775) [inline middleware/uploadMiddleware.js:45]: skąd ta liczba

File: middleware/uploadMiddleware.js [hunk 2]
```diff
@@ -33,4 +61,4 @@ const uploadStorage = multer.diskStorage({
     }
  });
 
- module.exports = { docsUpload };
\ No newline at end of file
+ module.exports = { docsUpload, profilePicUpload };
```
Comment (macibol775) [inline middleware/uploadMiddleware.js:64]: add a new line at the end of file

**Review by macibol775 — APPROVED** [review]
ok

**Review by WojciechPtas — APPROVED** [review]
LGTM

File: middleware/uploadMiddleware.js (hunk 1 above)
Comment (MasterCoookie) [reply middleware/uploadMiddleware.js:45]: 25 mb, z pizdy

**Review by bartorl — APPROVED** [review]
Rat likes

**Pull request merged**